Bio/PDB/parse_pdb_header.py

```diff
@@ -247,6 +296,9 @@ def _parse_pdb_header_list(header):
                 except:
                     # print('nonstandard resolution %r' % r)
                     dict['resolution'] = None
+            elif hh.startswith("REMARK 465"):
+                # Update the dictionary with content of the remark 465 line (Missing residues)
+                _parse_remark_465(tail, dict)
```

Comment:
Minor: I would personally prefer the syntax `dict.update(_parse_remark_465(tail))` - passing the result data structure doesn't feel very pythonic.